src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -319,6 +319,8 @@ sequences using different contrast enhanced images. The label is the name of the
 contrast agent. The key `ContrastBolusIngredient` MAY be also be added in the
 JSON file, with the same label.
 
+The OPTIONAL [`task-<label>`](../99-appendices/09-entities.md#task) entity can be used in order to allow tasks during structural MR acquisitions, e.g. pre-described motion paradigms such as nodding, to be described. Note, just as for functional imaging each task has a unique label that MUST only consist of letters and/or numbers (other characters, including spaces and underscores, are not allowed). Those labels MUST be consistent across subjects and sessions.
```

Comment:
I would recommend removing the reference to the functional imaging, the label is what it is, without reference to other entries which may or may not always remain consistent.

Additionally, I think it should be highlighted that variations in acquisition quality (be they extraneous sources of noise or just head movement) are specifics of acquisition and not tasks in the context of a biological response (more information [here](https://github.com/bids-standard/bids-specification/issues/1177#issuecomment-1205817585)).

Perhaps:

"""
[...] can be used in order to document tasks performed during anatomical MR acquisition, the effects of which may pertain to brain anatomy. Note that variations in acquisition quality, as produced by either the subject or extraneous sources, are acquisition parameters and not tasks. Each task has a unique label that MUST only consist of letters and/or numbers (other characters, including spaces and underscores, are not allowed). Those labels MUST be consistent across subjects and sessions.
"""

Reply:
> can be used in order to document tasks performed during anatomical MR acquisition, the effects of which may pertain to brain anatomy. Note that variations in acquisition quality, as produced by either the subject or extraneous sources, are acquisition parameters and not tasks. 

I think this definition is too limiting. And I'm not sure we should be so quick to define things such as quality; those definitions should be left up to the researcher/study as we have no idea what they're measuring.

More broadly defining tasks to be say an action/inaction (task for lack of a better word) that is performed by the subject would be my preference. Also, using `task-` in this manner allows for the use of all the accompanying features already incorporated within, e.g. events files.  Which to me makes even more of an argument for broadening task outside of `fmri`. see [here](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/05-task-events.html)

Reply:
I agree that subject actions requested by the experimenter should count as a task (e.g., resting-state scans). I still think events files should be required when the `task` entity is present though.